### DISCUSSION TITLE
Temporal Observable Specific Signal Biases

### DISCUSSION THREAD

**Discussion by Waley04** [groops-devs/groops#131] [Q&A] [answered]
Dear Developers,

I observed that GROOPS software can estimate signal biases. I was, therefore, wondering if you could advise on how I can estimate the temporal OSB (per PRN for example) for a given frequency.  May I know if [GnssSinexBias2SignalBias](https://groops-devs.github.io/groops/html/GnssSignalBias2SinexBias.html) supports this? 

Thanks for your support.

Regards,

Waley.

**Comment by tmayerguerr** [comment, marked as answer]
Dear Waley,

You can setup temporal changing signal bias parameters with [parametrization](https://groops-devs.github.io/groops/html/gnssParametrizationType.html)->TemporalBias in GnssProcessing. 

- The program [GnssSinexBias2SignalBias](https://groops-devs.github.io/groops/html/GnssSinexBias2SignalBias.html) do not support temporal biases.
- The program [GnssSignalBias2SinexBias](https://groops-devs.github.io/groops/html/GnssSignalBias2SinexBias.html) can handle Time-variable biases.

So, we can write out our estimations but we cannot use external information.

Hope this answers your question.

regards
Torsten
